power-policy-service/src/config.rs

```diff
@@ -10,6 +10,8 @@ pub struct Config {
     pub provider_unlimited: PowerCapability,
     /// Power capability of every provider in limited power mode
     pub provider_limited: PowerCapability,
+    /// Minimum power threshold to consume power from
```

Comment:
Probably worth describing the behavior if `None`
```suggestion
    /// Minimum power threshold to consume power from.
    ///
    /// If [`None`], the service will consume from providers, regardless of how much power they provide.
```

Reply:
Done.